src/cli/subcommands/state_cmd.rs

```diff
@@ -55,6 +64,28 @@ impl StateCommands {
                     .await?;
                 println!("{ret}");
             }
+            Self::ReadState { actor_address } => {
+                let tipset = ChainHead::call(&client, ()).await?;
+                let address = match StrictAddress::from_str(&actor_address) {
+                    Ok(address) => address.into(),
+                    Err(fvm_shared4::address::Error::UnknownNetwork) => {
+                        let expected = match CurrentNetwork::get() {
```

Comment:
I don't think `CurrentNetwork` reports the network of the node

Reply:
But we are using `CurrentNetwork::get()` inside the `StrictAddress::from_str()` to get the correct address prefix [here](https://github.com/ChainSafe/forest/blob/819df6fe041c73b5afe8fee47ffb5cc408fb1d16/src/shim/address.rs#L291). So I assumed it gives correct result.